### PR TITLE
Don't rely on validators having names when configuring pigeon

### PR DESCRIPTION
**Pull request by measure-fi**
Fixes https://github.com/palomachain/paloma/issues/310

## Background

Sometimes they don't, but they always have addresses.

## Testing completed

- [x] manual testing

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,7 +68,7 @@ Make sure your Paloma Cosmos-SDK keys are stored and available on your environme
 
 Set the VALIDATOR env variable
```
Comment (measure-fi) [inline README.md:69]: @Vizualni It is a little weird to me that we are setting `VALIDATOR` here, but _using_ `VALIDATOR` on the line above.

**Comment by measure-fi** [comment]
@Vizualni PTAL

File: README.md [hunk 2]
```diff
@@ -74,7 +74,7 @@ Make sure your Paloma Cosmos-SDK keys are stored and available on your environme
...
 
-`export VALIDATOR="$(palomad keys list --list-names | head -n1)"`
+`export VALIDATOR="$(palomad keys list --output json | jq -r '.[0].address')"`
```
Comment (Vizualni) [inline README.md:77]: variable VALIDATOR came from the paloma config where it's the name of the key in the keychain, so `VALIDATOR` so validator's address can't be used interchangeably in all places. Yes, it does work for some places, but not for all.

Comment (measure-fi) [reply README.md:77]: Ah. I would be okay renaming it. `VALIDATOR_ADDRESS`? The fact remains sometimes `palomad keys list --list-names` lists nothing.

**Comment by measure-fi** [comment]
@Vizualni 🏓 (we can also close this)

**Comment by Vizualni** [comment]
Closing (:

**Pull request closed without merging**